module/VuFind/tests/integration-tests/src/VuFindTest/Mink/LoggingTest.php

```diff
@@ -431,7 +465,6 @@ public function testNoEmailLoggingWhenDisabled(): void
         $emailLogPath = $this->getEmailLogPath();
         if (file_exists($emailLogPath)) {
             $loggedEmails = trim(file_get_contents($emailLogPath));
-            print_r($loggedEmails);
```

Comment:
This is extremely weird -- I see the print_r here in the pull request diffs, but it does not appear to be in the actual file!

Reply:
Never mind, I'm reading a "remove" as if it were an "add." Silly me!